docs/guides/run.md

```diff
@@ -57,6 +62,12 @@ ubuntu-hirsute         default                mypod2                 mypod2
 ubuntu-hirsute         default                mypod2                 mypod2                 242164  cat         0        0        3   /dev/null
 ```
 
+By default, it will use the `ghcr.io` as the registry, `inspektor-gadget/gadget` as the repository prefix and `latest` as the tag. You can also run them in fully qualified form as:
+
+```bash
+$ kubectl gadget run ghcr.io/inspektor-gadget/gadget/trace_tcpconnect:latest
+```
+
```

Comment:
I suppose this part is not needed anymore?

Reply:
Oops, thanks for catching it. Removed